axelor-account/src/main/java/com/axelor/apps/account/web/AccountingBatchController.java

```diff
@@ -265,4 +273,72 @@ public void printAccountingReport(ActionRequest request, ActionResponse response
   public void actionAccountingCutOff(ActionRequest request, ActionResponse response) {
     runBatch(AccountingBatchRepository.ACTION_ACCOUNTING_CUT_OFF, request, response);
   }
+
+  public void actionAutoMoveLettering(ActionRequest request, ActionResponse response) {
+    AccountingBatch accountingBatch = request.getContext().asType(AccountingBatch.class);
+    BatchAutoMoveLettering service = Beans.get(BatchAutoMoveLettering.class);
+    if (service.existsAlreadyRunning(accountingBatch)) {
+      response.setError(I18n.get(AccountExceptionMessage.BATCH_AUTO_MOVE_LETTERING_ALREADY_EXISTS));
+    }
+    boolean isMoveLinesAwaitingReconcile = service.getIsMoveLinesAwaitingReconcile(accountingBatch);
+    Boolean isShowLinesButtonDisplayed =
+        (Boolean) request.getContext().get("isShowMoveLinesInProposalBtnDisplayed");
+    if (isMoveLinesAwaitingReconcile
+        && (isShowLinesButtonDisplayed == null || !isShowLinesButtonDisplayed)) {
+      response.setFlash(
+          I18n.get(AccountExceptionMessage.BATCH_AUTO_MOVE_LETTERING_PENDING_PROPOSAL_EXISTS));
+      response.setValue("$isShowMoveLinesInProposalBtnDisplayed", true);
+    } else {
+      response.setValue("$isShowMoveLinesInProposalBtnDisplayed", false);
+      runBatch(AccountingBatchRepository.ACTION_AUTO_MOVE_LETTERING, request, response);
```

Comment:
add a response.setView to open a grid with the moveLines reconciled (even if isProposal = false)